go/api/commands.go

```diff
@@ -371,4 +371,18 @@ type StringCommands interface {
 	//
 	//[valkey.io]: https://valkey.io/commands/getdel/
 	GetDel(key string) (string, error)
+
+	// Del removes the specified keys. A key is ignored if it does not exist.
```

Comment:
Please copy docs from other clients to ensure that they are all aligned.
Add a cluster notice: https://github.com/valkey-io/valkey-glide/blob/783a9f09162926ef6dc308f5ebd73264f325544f/java/client/src/main/java/glide/api/commands/GenericBaseCommands.java#L46-L47